src/EmbeddedVsix/EmbeddedVsix.csproj

```diff
@@ -137,6 +134,17 @@
     </ItemGroup>
   </Target>
 
+  <PropertyGroup>
+    <VsixUtilPath>$(PkgMicrosoft_VSSDK_BuildTools)\tools\vssdk\bin\VsixUtil.exe</VsixUtilPath>
+    <CorFlagsPath>$(CORFLAGS_PATH)</CorFlagsPath>
+    <CorFlagsPath Condition="'$(CorFlagsPath)' == ''">corflags.exe</CorFlagsPath>
+  </PropertyGroup>
+  
+  <Target Name="ModifyVsixUtilCorFlags">
+    <Message Importance="high" Text="Modifying VsixUtil.exe CorFlags" />
+    <Exec Command="&quot;$(CorFlagsPath)&quot; &quot;$(VsixUtilPath)&quot; /32BIT- /force" />
```

Comment:
On my PC the flag is actually "/32BITREQ-", but the CI's version of corflags is different and has "/32BIT-" instead. I suppose we could modify our local versions manually as a workaround, or add a step that detects the version based on file's properties perhaps. WDYT?

Reply:
How big is corflags.exe?  I think it's standalone and not very big, so it might be easier to check in a copy under the `build` directory. That would be simpler and work on both CI and locally without changes.